docs/product-installation-guide/src/main/asciidoc/controller-eap-configure-proc.adoc

```diff
@@ -50,15 +50,7 @@ For more information, see <<eap-users-create-proc>>.
 The `org.kie.server.location` property specifies the location of {KIE_SERVER}.
 ====
 
-* Replace `<DECISION_CENTRAL_URL>` with the URL of {CENTRAL}. Decision Server connects to this URL during startup.
-+
-** If you installed {CENTRAL} using the installer or {EAP} zip installations, `<DECISION_CENTRAL_URL>` has this format:
-+
-`http://<HOST>:<PORT>/decision-central/rest/controller`
-** If you are running {CENTRAL} using the `standalone.jar` file, `<DECISION_CENTRAL_URL>` has this format:
-+
-`http://<HOST>:<PORT>/rest/controller`
-
+* Replace `<DECISION_CENTRAL_URL>` with the URL of the standalone {KIE_SERVER} Controller. Decision Server connects to this URL during startup. 
```

Comment:
Can you please add here an example of standalone Controller URL? So the users know how should it look like. The example URL is:
`
http://<HOST>:<PORT>/controller/rest/controller
`